chapters/annotations.tex

```diff
@@ -72,6 +72,19 @@ \subsection{Class Description and Revision History}\label{annotation-info-revisi
 <a href="modelica:/MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
 
+\subsubsection{Style sheet}
+
+A top-level package may contain an annotation \lstinline!styleSheet="modelica:/Package/Resources/style.css"!, which provides a cascading stylesheet.
+The given stylesheet is used when displaying the \lstinline!info! and \lstinline!revisions! annotations found in the \lstinline!Documentation! annotations.
+The provided stylesheet shall not use type or universal selectors.
```

Comment:
Two words:
```suggestion
The provided style sheet shall not use type or universal selectors.
```

Reply:
Is it really clear enough to our audience what _type or universal selectors_ mean?  Perhaps we could have a tiny inline example for each?
```suggestion
The provided stylesheet shall not use type (for example \lstinline[language=css]!h1 { $\ldots$ }!) or universal (for example \lstinline[language=css]!* { $\ldots$ }!) selectors.
```

Reply:
Hm... This made me realize that libraries do use (and want to use) type selectors, e.g. to modify the font (size, family, weight) of headings like h1 - and even universal selectors.
I'm not saying that it is necessarily a good idea, but neither do I think that we should forbid it outright (could make it up to tool-vendor?).